### ISSUE TITLE
[FEATURE REQUEST] Highlighting on curators' lists

### ISSUE THREAD

**Issue by kirara** [IsThereAnyDeal/AugmentedSteam#1356]
### Current Behavior (if any)
On curators' lists (e.g. https://store.steampowered.com/curator/6856591-Designer-Plays/list/71726) only original badges are showing.

### Desired Behavior
Highlight Owned and Wishlisted as anywhere else.

### Motivation / Use Case for Changing the Behavior
I'd like the lists to be more visually organized.

**Comment by candela97** [comment]
Yeah we need to add support for these "new style" pages including sales pages, the new store hubs (https://store.steampowered.com/news/group/35143931/view/3030352531883893693) etc. But so far I'm having trouble querying the nodes to process, because there're just so many moving parts that makes it difficult to handle with mutation observers. Thoughts? @MxtOUT
